skills/projectCommand.js

```diff
@@ -0,0 +1,18 @@
+const Project = require('../lib/project.js');
+const AddLinkToProjectAction = require('../lib/actions/addLinkToProjectAction.js');
+const Actions = require('../lib/actions/actions.js');
+const AddLinkToProjectResponse = require('../lib/actions/AddLinkToProjectResponse.js');
+
+
+module.exports = function(controller) {
+  controller.on('slash_command', async function (bot, message) {
+    if(message.command === '/project'){
+        let findChannel = Project.findByChannel(message.channel_id);
```

Comment:
I believe `findByChannel` is `async`/returns a `Promise`. If so, this might not be working as you intended.

Reply:
Oh I understand, we need to use await: ```let findChannel = await Project.findByChannel(message.channel_id);``` Correct?

Reply:
Done.